CodeHub/ViewModels/SearchViewmodel.cs

```diff
@@ -378,7 +378,7 @@ public void IssueNavigate(object sender, ItemClickEventArgs e)
             string owner = array[3];
             string repo = array[4];
 
-            SimpleIoc.Default.GetInstance<IAsyncNavigationService>().NavigateAsync(typeof(IssueDetailView),"Issues", new Tuple<string, string, Issue>(owner, repo, e.ClickedItem as Issue));
```

Comment:
@aalok05 Is this a typo?

Reply:
Oh yes that's a mistake. It should be 'Issue'.